web/src/main/java/com/epam/rft/atsy/web/mapper/RuleValidationExceptionMapper.java

```diff
@@ -1,8 +1,20 @@
 package com.epam.rft.atsy.web.mapper;
 
+/**
```

Comment:
The javadocs are without line breaks after the summary sentence because of the current code formatting conventions that we applied to this project.